### ISSUE TITLE
Do logit for binary outcomes? 

### ISSUE THREAD

**Issue by amarchenko26** [amarchenko26/metoo#60]
Do we need to do logits for binary variables like settlement and winning?
